Resources/Prototypes/_Mono/Entities/SpaceArtillery/sink_ports.yml

```
@@ -0,0 +1,4 @@
+- type: sinkPort
+  id: SpaceArtilleryFire
+  name: "Fire Armament"
+  description: "Controls Space Artillery's firing mechanism."
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)